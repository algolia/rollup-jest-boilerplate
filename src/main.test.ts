import rollupJestBoilerplate from './main';

describe('rollupJestBoilerplate', () => {
  it('rollupJestBoilerplate(string)', () => {
    expect(rollupJestBoilerplate('cool')).toMatchSnapshot();
  });
});
