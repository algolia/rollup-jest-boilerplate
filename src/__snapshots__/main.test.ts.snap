// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rollupJestBoilerplate rollupJestBoilerplate(string) 1`] = `
Object {
  "awesomeString": "cool",
}
`;
